lib/index.js

```diff
@@ -98,7 +98,16 @@ export default {
           return null;
         }
 
-        const fileText = editor.getText();
+        const removePHP = str => str.replace(/<\?php(?:[\s\S])+?\?>/gi, (match) => {
+          const newlineCount = (match.match(/\r?\n|\r/g) || []).length;
```

Comment:
You shouldn't need the second `|\r` here.

Reply:
Instead of creating an empty array and getting the length of that, do `match...length || 0`.

Reply:
`/\r?\n|\r/g` matches LF (`\n`), CR (`\r`) and CR+LF (`\r\n`),
I'll fix the `.length` issue.